app/assets/javascripts/services/notification_svc.js

```diff
@@ -1,6 +1,7 @@
 samson.service('notificationService', function(messageCenterService) {
-  this.info = function(message, options) {
+  var _this = this;
```

Comment:
True. Was trying to remember the exact convention for that... :-P

I'm going to change that.